packages/graph/hash_graph/lib/graph/src/store/postgres/query/entity_type.rs

```diff
@@ -48,7 +48,7 @@ impl Path for EntityTypeQueryPath {
                         column: "target_property_type_version_id",
                     },
                     join_table_name: TableName::PropertyTypes,
-                    join_column_access: DataTypeQueryPath::VersionId.column_access(),
```

Comment:
Was this just a typo/mistake before? :)

Reply:
Yes. The result is the same, but using `PropertyTypeQueryPath` is clearer as the `Table` also is `PropertyTypes`. In the long run we should probably also have enumerations for columns instead, so we can use something like `PropertyTypesColumn::VersionId`.